src/apps/FEMSWEEP.cpp

```diff
@@ -0,0 +1,141 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "FEMSWEEP.hpp"
+#include "FEMSWEEP_DATA.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include "common/DataUtils.hpp"
+
+#include <algorithm>
+#include <cmath>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+
+FEMSWEEP::FEMSWEEP(const RunParams& params)
+  : KernelBase(rajaperf::Apps_FEMSWEEP, params)
+{
+  m_ne = 15 * 15 * 15;
```

Comment:
Are there parameter names associated with the numbers choices? For example, does 15 mean anything?

Reply:
I can leave some comments here clarifying that 15 is the number of elements per dimension for a cube. However, this will be subject to change once we have a mesh generator. The geometry may not be a cube in the future.

Reply:
Why not make class member variables such as `m_nx`, `m_ny`, `m_nz` and avoid comments?